bin/metadata.js

```diff
@@ -28,11 +27,11 @@ async function main(prid, owner, repo, logger) {
   const metadata = new MetadataGenerator(data).getMetadata();
   const [SCISSOR_LEFT, SCISSOR_RIGHT] = MetadataGenerator.SCISSORS;
   logger.info({
-    raw: [SCISSOR_LEFT, metadata, SCISSOR_RIGHT].join(EOL)
+    raw: `${SCISSOR_LEFT}${metadata}${SCISSOR_RIGHT}`
```

Comment:
While I understand the intent to prevent too much data manipulation inside the bin, I'm not fond of this particular change, I find it less readable. But maybe that's because I'm less experienced.

Reply:
It's just a template literal: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals — there should be no need to create an array for something so basic.

Reply:
Well, the template litteral I know of, but tbh, at first reading I was wondering how you were gonna include the EOL., that's the only thing that's less readable to me, I thought it was going to be single line ^^

Reply:
@Tiriel it is included in `MetadataGenerator.SCISSORS` in file `lib/metadata_gen.js` line 49-50.

Reply:
@cPhost Yeah I know, I saw that after, but that's not very explicit or more readable. That's the only thing I'm saying. I fail to see why it's better that way, that's all ^^

Reply:
That probably has more to do with the fact that the EOL used to be explicitly included in this file. I would expect something like SCISSORS and metadata to already include EOL.

Reply:
Having EOL declared in this file means that when we want to use metadata or scissors in another target (such as `land-pr`) we need to remember that they don't include EOL and include it there. The more targets, the more times we have to duplicate that code and the more chances that an inadvertent bug sneaks in.

Reply:
Also, if you just think about the metadata content conceptually, it's not complete without the final EOL — that final EOL is something that is needed to make it complete.

Reply:
I see your point!

Thanks for taking the time!